packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageSearchBar.js

```diff
@@ -18,57 +18,58 @@ import * as React from 'react';
 import { Button, Input } from 'antd';
 
 import * as markers from './TracePageSearchBar.markers';
+import UiFindInput from '../../common/UiFindInput';
 
 import './TracePageSearchBar.css';
 
 type TracePageSearchBarProps = {
-  updateTextFilter: string => void,
   textFilter: string,
   prevResult: () => void,
   nextResult: () => void,
   clearSearch: () => void,
   resultCount: number,
   forwardedRef: { current: Input | null },
+  traceGraphView: boolean,
```

Comment:
nice name. I was just passing down the `TracePageHeader` prop but definitely better to name `TracePageSearchBar`'s prop so that it's relevant to that component 👍